specification/sql/resource-manager/Microsoft.Sql/preview/2024-05-01-preview/ManagedInstanceLongTermRetentionPolicies.json

```diff
@@ -0,0 +1,364 @@
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          },
+          "202": {
+            "description": "Accepted",
+            "headers": {
```

Comment:
why are we adding locations in header, can we add relevant example.

Reply:
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers

Please refer this for guidelines